translations/nl-be.json

```diff
@@ -526,5 +529,15 @@
   "search-legend-not-explanation": "NIET: Zoek op 'Antwerpen' en neem alle resultaten waar ook 'verkeer' in zit, weg",
   "search-legend-order-explanation": "Zoekt op de gehele reeks van woorden in die volgorde",
   "search-legend-wildcard-explanation": "Zoeken waarbij het opgegeven lemma de prefix is voor een woord",
-  "submitted-by": "- Indiener"
+  "submitted-by": "- Indiener",
+  "no-themes-selected-message": "Er zijn nog geen thema's geselecteerd. Bent u zeker dat u het kort bestek wilt opslaan?",
+  "manage-users-name": "Naam",
```

Comment:
Is this the first name, last name or the full name? 
I assume the last option so the correct term would be "manage-users-full-name"
Or just make it general and name it "full-name"

Reply:
Changed it to fullname